SDK/ShopItem.php

```diff
@@ -248,10 +262,12 @@ protected function loadImage($path)
 			if ($image_data === false)
 				throw new Exception('Image does not exists');
 
+			$mime_type = $this->getMimeType($path);
+
 			//correct image
 			if (Tools::strlen($image_data) > 0)
 			{
-				$image_data = $this->resize($image_data, 300, 300);
+				$image_data = $this->resize($image_data, 300, 300, false, $mime_type);
 				$a = 'base64_encode';
```

Comment:
we need base64_encode to pass the image of the product to the API server in JSON. So if you're fine using base64_encode for this purpose, we're fine removing this workaround.